modules/foreman_compute_profile.py

```diff
@@ -65,6 +68,12 @@
 
 RETURN = ''' # '''
 
+from ansible.release import __version__ as ansible_version
+if [int(i) for i in ansible_version.split(".")] >= [2, 3, 0]:
+    HAS_PROPER_VERSION = True
+else:
+    HAS_PROPER_VERSION = False
+
```

Comment:
There is a logical error in this: When used with ansible < 2.3, the subsequent import of foreman_helper will still fail with an exception before we can report the incompatibility.

Reply:
yes, thanks :) I did not finish the PR yet, hence WIP ;)